stl/inc/type_traits

```diff
@@ -521,12 +521,14 @@ _INLINE_VAR constexpr bool is_volatile_v<volatile _Ty> = true;
 template <class _Ty>
 struct is_volatile : bool_constant<is_volatile_v<_Ty>> {};
 
+#if !_HAS_DEPRECATED_TYPETRAITS

```

Comment:
As @AdamBucior observed in https://github.com/microsoft/STL/pull/380#issuecomment-566749706 , `is_pod` wasn't removed from C++20.

Reply:
Done